lib/safira_web/views/attendee_view.ex

```diff
@@ -28,7 +28,7 @@ defmodule SafiraWeb.AttendeeView do
       avatar: Avatar.url({attendee.avatar, attendee}, :original),
       badges: render_many(attendee.badges, SafiraWeb.BadgeView, "badge.json"),
       badge_count: attendee.badge_count,
-      volunteer: attendee.volunteer,
+      # volunteer: attendee.volunteer,
```

Comment:
Since the field it's being removed. It makes all the sense to delete this line instead of commenting it. 
**Same for all the others.**

Reply:
Just did it.